lib/Doctrine/ORM/Tools/Console/Command/RunDqlCommand.php

```diff
@@ -120,7 +120,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $query->setMaxResults((int) $maxResult);
         }
 
-        if ($input->hasOption('show-sql')) {
+        if ($input->hasOption('show-sql') && $input->getOption('show-sql')) {
```

Comment:
Is that a case we should be even thinking about? Why would anyone remove an option?